Simplified.xcodeproj/project.pbxproj

```diff
@@ -141,7 +141,6 @@
 		1744DF4E2640B31B00CBAAA8 /* Stringable.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1744DF492640B31A00CBAAA8 /* Stringable.swift */; };
 		175D76D6264DAE8500BD11BA /* bpl_authentication_document.json in Resources */ = {isa = PBXBuildFile; fileRef = 175D76D5264DAE8500BD11BA /* bpl_authentication_document.json */; };
 		175D76D7264DAE8500BD11BA /* bpl_authentication_document.json in Resources */ = {isa = PBXBuildFile; fileRef = 175D76D5264DAE8500BD11BA /* bpl_authentication_document.json */; };
-		175D77752655F55A00BD11BA /* OPDS2LibraryRegistryFeed.json in Resources */ = {isa = PBXBuildFile; fileRef = B51C1DFB22860513003B49A5 /* OPDS2LibraryRegistryFeed.json */; };
```

Comment:
This and the line below should be removed from the last PR but I forgot, doing it here since it doesn't worth another PR